packages/format-currency/package.json

```
@@ -0,0 +1,27 @@
+{
+	"name": "format-currency",
```

We've been trying to keep these in the `@automattic` scope. Any objections to continuing that?
```suggestion
	"name": "@automattic/format-currency",
```

---

Also, [it's taken](https://www.npmjs.com/package/format-currency) 😂